product_pricelist_item_formula_visibility/views/product_pricelist_item_views.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <record id="product_pricelist_item_tree_view_inherit" model="ir.ui.view">
```

Comment:
Remember also adding these columns in pricelist item list view within pricelist form (view `product.product_pricelist_view`)